docs/pages/branding/pricing.tsx

```diff
@@ -0,0 +1,707 @@
+              imgProps={{ width: 36, height: 36 }}
+              title="Perpetual license"
+            >
+              <Box sx={{ mt: 2 }}>
+                When you purchase,{' '}
+                <strong>
+                  you are granted a license to use a version of the product in perpetuity
+                </strong>
+                . There are no further charges unless you choose to extend your license to cover
+                newer versions.
+                <br />
+                <br />
+                Please note that while the use of the software is perpetual, support and corrective
+                maintenance are not. We do not provide issue resolution to versions older than 12
+                months. We roll bug fixes, performance enhancements, and other improvements into new
+                releases; we don&apos;t patch, fix or in any way alter older versions.
+              </Box>
+            </BrandingCard>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <BrandingCard
+              image="/static/branding/pricing/community.svg"
+              imgProps={{ width: 40, height: 40 }}
+              title="1-year subscription to new versions"
+            >
+              <Box sx={{ mt: 2 }}>
+                When you make a purchase{' '}
+                <strong>you get a subscription to license new versions for 366 days</strong>. You
+                can see our{' '}
+                <Link href="https://github.com/mui-org/material-ui-x/releases">changelog</Link>.
+                After 366 days (or up to five-year if you choose an extension package) you will no
+                longer be allowed to update to the latest versions. You can continue to use your
+                licensed versions in perpetuity.
+                <br />
+                <br />
+                Please note that while the use of the software is perpetual, support and corrective
+                maintenance are not. We roll bug fixes, performance enhancements, and other
+                improvements into new releases; we don&apos;t patch, fix or in any way alter older
+                versions.
+              </Box>
+            </BrandingCard>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <BrandingCard
+              color="emerald"
+              image="/static/branding/pricing/documentation.svg"
+              imgProps={{ width: 40, height: 32 }}
+              title="Subscription renewal"
+            >
+              <Box sx={{ mt: 2 }}>
+                While the use of the software is perpetual, access to new features, support and
+                corrective maintenance are not. At the end of your subscription period, you will
+                need to renew your license to access updates and support.{' '}
+                <b>Renewal pricing is 50% than first-year subscription costs.</b> The renewal can
+                range from 366 days up to a five-year term.
```

Comment:
```suggestion
                range from one to five years.
```

But the initial purchase can't?